tests/unit/test-cmd/test.run.js

```diff
@@ -127,6 +128,18 @@ describe('run', () => {
     });
   });
 
+  it('passes --jsconsole when specified', () => {
+    const firefox = '/pretend/path/to/Firefox/firefox-bin';
+    const cmd = prepareRun();
+    const {firefoxApp} = cmd.options;
+
+    return cmd.run({firefox}).then(() => {
+      assert.equal(firefoxApp.run.called, true);
+      assert.equal(firefoxApp.run.firstCall.args[1].binaryArgs,
+              '-jsconsole');
```

Comment:
when doing a vertical alignment, always line up to the inside of the opening parenthesis, like:
````javascript
      assert.equal(firefoxApp.run.firstCall.args[1].binaryArgs,
                   '-jsconsole');
````